apps/launchpad/app/lib/graphql/types.ts

```diff
@@ -1,11 +1,14 @@
 import { gql } from 'graphql-request'
 
 export interface Question {
+  id: number
   title: string
   description: string
   enabled: boolean
   point_award_amount: number
-  id: string
+  link: string
+  predicate_id: number
```

Comment:
Good idea -- I don't remember if we talked about this or not but I was thinking we'd add this to the API as well.